README.md

```
@@ -13,14 +13,15 @@
 
 Install all dev tools
 ```shell
+mise upgrade --local --bump --interactive
 mise i
 ```
 
-Install all deps if not done by `mise`
+Install all deps if not done by `mise` add `sfw` beforehand
 ```shell
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)